app/server/express.js

```diff
@@ -27,91 +27,13 @@ const tokenRouterV1 = require("./routes/v1/token");
 
 const authenticateToken = require("./middleware/authenticateToken");
 
-const swaggerOptions = {
```

Comment:
As I understand it, you've abstracted this out to a dedicated file. Is this correct?